script/Ciel/Day 15/15_01B_CIEL15B_1.txt

```diff
@@ -2274,13 +2277,13 @@ All I ever dream of are the events of that night, and the things I got up to bef
 -- 僕の現実、僕が知り得た知識、そこから想像できるであろう未来は、とても狭くて救われないものばかりだから、そういったものを想像する事さえできない」
 My reality,
  my acquired knowledge,
- and the futures I can picture based on that knowledge are all so small and bereft of salvation that I can't even conceive of such a thing."
+ and the futures I can picture based on that knowledge are all so small and without salvation that I can't even conceive of such a future."
 }
 [sha:e04834567208967fe10ec9a85a92f549dece46a0]{
 -- Page 140, Offset 42374.
 -- 「けど君のユメは違った。君にとっては当たり前の風景も、僕にとってみれば輝かしい風景なんだ。それこそ夢にまで見たっていうヤツだよ。
 "But your dreams are different.
- Those scenes may appear ordinary to you, but to me, they're so brilliant they're almost blinding.
+ Those scenes may appear ordinary to you, but to me, they're so radiant with happiness it's almost blinding.
```

Comment:
Reifying the happiness maybe putting some words in his mouth? Also might restructure it  a bit for flow.
> Those scenes may appear ordinary to you, but to me, 
> they're so happy I'm almost blinded by their radiance.

Reply:
hmmm, I think it's true that it might not be happiness
not sure what I was thinking here, but I might've been too stuck on the talks of happiness from before
I'll revert this